lib/dstu2_resources.rb

```diff
@@ -2786,99 +2786,251 @@ module Resources
-            "resource": {
-              "resourceType": "Encounter",
-              "id": "222",
-              "text": {
-                "status": "generated",
-                "div": "&lt;div>&lt;p>&lt;b>Encounter&lt;/b>&lt;/p>&lt;p>&lt;b>Location&lt;/b>: BW Rheumatology&lt;/p>&lt;p>&lt;b>Type&lt;/b>: outpatient&lt;/p>&lt;p>&lt;b>Status&lt;/b>: finished&lt;/p>&lt;p>&lt;b>Period Start Date&lt;/b>: 2015-03-12T15:30:00.000Z&lt;/p>&lt;p>&lt;b>Period End Date&lt;/b>: 2015-03-12T16:15:00.000Z&lt;/p>&lt;p>&lt;b>Reason For Visit&lt;/b>: rheumatoid arthritis&lt;/p>&lt;p>&lt;b>Attending Physician&lt;/b>: Jones, Bob&lt;/p>&lt;/div>"
-              },
-              "identifier": [
+            "resourceType": "Location",
+            "id": "d312719",
+            "type": {
+              "coding": [
                 {
-                  "use": "usual",
-                  "system": "http://cerner.com/fhir/identifier/FINNBR",
-                  "value": "20001118"
+                  "system": "http://hl7.org/fhir/v3/RoleCode",
+                  "code": "PTRES",
+                  "display": "Patient's Residence",
+                  "userSelected": false
                 }
               ],
-              "status": "finished",
-              "class": "outpatient",
-              "patient": {
-                  "reference": "Patient/111"
-              },
-              "participant": [
+              "text": "Home"
+            }
+          },
+          {
+            "resourceType": "Location",
+            "id": "o589723",
+            "name": "Baseline West Medical Center",
+            "type": {
+              "coding": [
                 {
-                  "type": [
-                    {
-                      "coding": [
-                        {
-                          "system": "http://hl7.org/fhir/v3/ParticipationType",
-                          "code": "ATND",
-                          "display": "attender"
-                        }
-                      ],
-                      "text": "attender"
-                    }
-                  ],
-                  "individual": {
-                    "reference": "Practitioner/4340404",
-                    "display": "Jones, Bob"
+                  "system": "http://hl7.org/fhir/v3/RoleCode",
+                  "code": "ER",
+                  "display": "Emergency room",
+                  "userSelected": false
+                }
+              ],
+              "text": "Emergency Room - Hospital"
+            }
+          }
+        ],
+        "identifier": [
+          {
+            "use": "usual",
+            "type": {
+              "text": "FIN NBR"
+            },
+            "value": "200124881",
+            "period": {
+              "start": "2016-02-15T18:57:37.000Z"
+            }
+          }
+        ],
+        "status": "in-progress",
+        "class": "outpatient",
+        "type": [
+          {
+            "text": "Emergency"
```

Comment:
As discussed offline, we currently do not have any mappings defined for Encounter.type and a discussion on an appropriate binding is yet inprogress. Removed the terminology bindings and added a note that Encounter.type is currently not bound to any terminology.
https://github.com/cerner/fhir.cerner.com/pull/69/commits/acb087d77b5e47073e028746894d04295c1fcc48